includes/api/class-wc-admin-rest-reports-coupons-controller.php

```diff
@@ -104,7 +100,7 @@ public function get_items( $request ) {
 	 * @return WP_REST_Response
 	 */
 	public function prepare_item_for_response( $report, $request ) {
-		$data = get_object_vars( $report );
+		$data = $report;
```

Comment:
It's done this way because the filter further down needs the original `$report` object, while other functions in between can change the response data.

Reply:
Ah, I see it now!  My mistake.